R/method-register.R

```diff
@@ -93,7 +93,7 @@ register_method <- function(generic,
   invisible(generic)
 }
 
-register_S3_method <- function(generic, signature, method) {
+register_S3_method <- function(generic, signature, method, envir = parent.frame()) {
```

Comment:
Call this `env` for consistency with S7 argument names?

Reply:
Happy to rename that